.github/workflows/publish-release-artifact.yml

```
@@ -29,10 +29,17 @@ jobs:
           PROJECT_VERSION=${{ steps.set-version.outputs.project-version }}
           echo $PROJECT_VERSION
           echo "file-name=$FILE_NAME" >> $GITHUB_OUTPUT
+          SHA256SUM=$(curl -fsSL https://repo.jenkins-ci.org/api/storage/releases/$GROUP_ID/$ARTIFACT_ID/$PROJECT_VERSION/$ARTIFACT_ID-$PROJECT_VERSION.jar.sha256)
```

this returns 404 when I try run it locally in my shell.

when I change it to this it works:

```suggestion
          SHA256SUM=$(curl -fsSL https://repo.jenkins-ci.org/releases/$GROUP_ID/$ARTIFACT_ID/$PROJECT_VERSION/$ARTIFACT_ID-$PROJECT_VERSION.jar.sha256)
```

---

Thanks, this is what I meant to commit in 1628a754131f92066d624f4fa1d1be59f472034a.